### DISCUSSION TITLE
v0.7.3-beta1

### DISCUSSION THREAD

**Discussion by kaloudis** [ZeusLN/zeus#1316] [Announcements]
## v0.7.3 Highlights
* QRCodeScanner: import QR code images from gallery 
* Biometrics login
* Channels: view pending and closed
* Channels: force close channel timer display
* Fast node switching
* LNURL-auth: LNDHub: compatibility mode for Alby and BlueWallet 

## What's Changed
* Bug fix: ZEUS-1260: routing fee totals being multiplied by @Talej in https://github.com/ZeusLN/zeus/pull/1281
* Clipboard: handleAnything cleanup by @Talej in https://github.com/ZeusLN/zeus/pull/1291
* POS: Reconciliation by @kaloudis in https://github.com/ZeusLN/zeus/pull/1294
* Unified Invoice: don't display until both on-chain + LN invoices are loaded by @kaloudis in https://github.com/ZeusLN/zeus/pull/1308
* QRCodeScanner: import QR code images from gallery by @shubhamkmr04 in https://github.com/ZeusLN/zeus/pull/1296
* c-lightning-REST: Fix wrong `getChannels` endpoint by @sha-265 in https://github.com/ZeusLN/zeus/pull/1266
* WalletHeader: SettingsButton: go to Nodes on long-press for fast switching by @kaloudis in https://github.com/ZeusLN/zeus/pull/1309
* Reproducible builds: update Docker image, use SHA256 hashes by @kaloudis in https://github.com/ZeusLN/zeus/pull/1310
* LNURL-pay: confirm button: use theme colors by @InnocuousFinch in https://github.com/ZeusLN/zeus/pull/1313
* Channels: force close channel timer display by @Talej in https://github.com/ZeusLN/zeus/pull/1292
* Pending/closed channels by @kaloudis in https://github.com/ZeusLN/zeus/pull/1019
* Images: add Lottie file support, new loading + payment indicators by @kaloudis in https://github.com/ZeusLN/zeus/pull/1314
* LNURL-auth: LNDHub: compatibility mode for Alby and BlueWallet by @Talej in https://github.com/ZeusLN/zeus/pull/1305
* Biometrics login by @channelninja in https://github.com/ZeusLN/zeus/pull/1301

## New Contributors
* @sha-265 made their first contribution in https://github.com/ZeusLN/zeus/pull/1266
* @channelninja made their first contribution in https://github.com/ZeusLN/zeus/pull/1301

**Full Changelog**: https://github.com/ZeusLN/zeus/compare/v0.7.2...v0.7.3-beta1

## External links

[PGP Key](https://zeusln.app/PGP.txt)

### Android
[APK](https://zeusln.app/zeus-v0.7.3-beta1.apk)
[APK Signature](https://zeusln.app/zeus-v0.7.3-beta1.apk.asc)
[Signed Manifest](https://zeusln.app/manifest-v0.7.3-beta1.txt)

### iOS
[TestFlight](https://testflight.apple.com/join/gpVFzEHN)

<hr /><em>This discussion was created from the release <a href='https://github.com/ZeusLN/zeus/releases/tag/v0.7.3-beta1'>v0.7.3-beta1</a>.</em>

**Comment by sha-265** [comment]
Maybe we should mention somewhere that supported version of CLN-REST is >= v0.10.0
